salt/android-sdk/init.sls

```diff
@@ -0,0 +1,21 @@
+{% set version_id = '3859397 sha256=444e22ce8ca0f67353bda4b85175ed3731cae3ffa695ca18119cbacef1c1bea0' %}
+{% set version, source_hash = version_id.split() %}
+
+android-sdk-deps:
+    pkg.installed:
+        - name: unzip
+
+android-sdk:
+    file.managed:
+        - name: /usr/local/src/sdk-tools-linux-{{ version }}.zip
+        - source: https://dl.google.com/android/repository/sdk-tools-linux-{{ version  }}.zip
+        - source_hash: {{ source_hash }}
+
+    cmd.watch:
+        - name: unzip -d /opt/android-sdk /usr/local/src/sdk-tools-linux-{{ version }}.zip &&
+                find android-sdk -type f -exec chmod 644 {} \; &&
```

Comment:
The paths given to find here doesn't seem to align with where stuff is unpacked. Is the permission modification necessary here at all?

Reply:
Actually, this probably breaks the sdk as it removes the executable bit from the binaries. But I see that as it's shipped the binaries are only executable by root, where the intention is probably that they should be executed by everyone. Maybe find existing executable binaries and make them 755 too?